grails-app/utils/org/broadinstitute/orsp/utils/UtilityClass.groovy

```diff
@@ -83,7 +85,27 @@ class UtilityClass {
                         reviewCategory: StringUtils.isNotEmpty(reviewCategory) ? reviewCategory : '',
                         reporter       : issue.reporter,
                         requestDate    : issue.requestDate,
-                        attachments    : issue.attachments
+                        attachments    : issue.attachments,
+                        actor          : issue.getActorUsernames()
+                ]
+            }
+        }
+    }
+
+    static void registerQaReportIssueMarshaller() {
+        JSON.createNamedConfig(ISSUE_FOR_QA) {
+            it.registerObjectMarshaller( StatusEventDTO ) { StatusEventDTO statusEvent ->
+                return [
+                    id             : statusEvent.issue.id,
+                    type           : statusEvent.issue.type,
+                    projectKey     : statusEvent.issue.projectKey,
+                    summary        : statusEvent.issue.summary,
+                    status         : statusEvent.issue.approvalStatus == IssueStatus.Legacy.name ? statusEvent.issue.status : statusEvent.issue.approvalStatus,
```

Comment:
The fact that this ternary needs to exist is concerning. We have two different statuses that are sort of the same thing ... sometimes. This is super confusing for any dev to have to reason about and will cause some bugs as folks try to remember which status to show in which case. Instead, we should have a single place where we get the status that does this kind of check for us. You can override this in `Issue` with this logic with a nice comment explaining why this problem exists ... and hopefully a link to the ticket that fixes this across the board.

Reply:
@rushtong  I totally agree with you and we just discuss this with Leo. We will create a ticket to add this logic in `Issue` (and also add a comment with the explanation of why we need this) and delete this ternary from all the places we are using it. Sounds good?

Reply:
Sounds good, thank you!

Reply:
@rushtong  https://broadinstitute.atlassian.net/browse/BTRX-776